lxd/resource_lxd_profile.go

```diff
@@ -209,3 +227,30 @@ func resourceLxdProfileExists(d *schema.ResourceData, meta interface{}) (exists
 
 	return
 }
+
+func resourceLxdProfileImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	p := meta.(*LxdProvider)
+
+	ids := strings.SplitN(d.Id(), ":", 2)
+	name := ids[0]
+	d.SetId(name)
+
+	if len(ids) == 2 {
+		d.Set("remote", ids[1])
+	}
```

Comment:
Is it correct to say that in order to import the profile, you must do:

```shell
$ terraform import lxd_profile.profile_name profile_name:remote_name
```

Would it make more sense to reverse the import name and do:

```shell
$ terraform import lxd_profile.profile_name remote_name:profile_name
```

?

I think you can use [`conf.ParseRemote`](https://github.com/lxc/lxd/blob/daff16273d9ff353ed30bbecd9d368555111c341/lxc/config/remote.go#L20) to easily split the name.

Reply:
mmmm maybe in this scope it makes sense to have the optional argument as prefix, because the remote  is optional, i'll make a change for this later this week

Reply:
[remote_name:]profile_name makes more sense to me too! 👍